sonar-plugin-api/src/main/java/org/sonar/api/resources/Project.java

```diff
@@ -34,7 +34,9 @@
  * A class that manipulates Projects in the Sonar way.
  *
  * @since 1.10
+ * @deprecated since 5.6 should not be used in any API
  */
+@Deprecated
 public class Project extends Resource implements Component {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)